plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -160,7 +156,7 @@ export class EventPipelineRunner {
     }
 
     private async handleError(err: any, currentStepName: string, currentArgs: any, sentToDql: boolean) {
-        const serializedArgs = currentArgs.map((arg: any) => this.serialize(arg))
+        const serializedArgs = currentArgs
         status.error('🔔', 'step_failed', { currentStepName, err })
         Sentry.captureException(err, { extra: { currentStepName, serializedArgs, originalEvent: this.originalEvent } })
         this.hub.statsd?.increment('kafka_queue.event_pipeline.step.error', { step: currentStepName })
```

Comment:
```suggestion
        status.error('🔔', 'step_failed', { currentStepName, err })
        Sentry.captureException(err, { extra: { currentStepName, currentArgs, originalEvent: this.originalEvent } })
        this.hub.statsd?.increment('kafka_queue.event_pipeline.step.error', { step: currentStepName })
```

Or something like that

Reply:
I was being lazy :D